src/main/java/Duke.java

```diff
@@ -1,10 +1,128 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static void initialiseWelcomeMessage() {
```

Comment:
I like how your methods name are self-explanatory. I see this in other places as well.

Reply:
Thank you! I will take note and keep noticing on that! :)